### PR TITLE
#143865383 Fix failing tests and update travis

### PR DESCRIPTION
**Pull request by andela-onnenanya**
#### What does this PR do?
Fix failing tests and update travis.yml
#### Description of Task to be completed?
Fix failing tests and update travis.yml
#### How should this be manually tested?
Run 'npm test' or open travis-ci dashboard for the branch
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Fix failing tests and update travis.yml
#### Screenshots (if appropriate)
screen shot 2017-04-20 at 1 36 49 am

image
#### Questions:
N/A

### CODE REVIEW

File: article.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Module dependencies.
+ */
+var mongoose = require('mongoose'),
```
Comment (emmyakin) [inline article.js]: this file should be written in es6 syntax

File: test/user/model.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ describe('<Unit Test>', function() {
         });
 
         describe('Method Save', function() {
```
Comment (emmyakin) [inline test/user/model.js]: you should also write this is es6 syntax arrow functions

**Review by emmyakin — CHANGES_REQUESTED** [review]

File: .eslintrc [hunk 3]
```diff
@@ -14,6 +14,7 @@
       "no-constant-condition": 1,
       "one-var-declaration-per-line": 0,
       "new-cap": 0,
+      "no-console": "off",
```
Comment (emmyakin) [inline .eslintrc]: can you remove this.
if you need to disable console.log, where needed, use // eslint-disable-line

File: test/frontend/unit/testingAngularUnitSpec.js [hunk 4]
```diff
@@ -1,14 +1,13 @@
...
   describe('Testing header Controller', function () {
-    it('should test scope object', function () {
+    it('should test scope object', function() {
       module('mean.system');
-      var scope = {};
-      var ctrl;
+      const scope = {};
       inject(function ($controller) {
-        ctrl = $controller('HeaderController', { $scope: scope, Global: { test: 'test' } });
+        const ctrl = $controller('HeaderController', { $scope: scope, Global: { test: 'test' } });
```
Comment (emmyakin) [inline test/frontend/unit/testingAngularUnitSpec.js]: use // eslint-disable-line here

File: test/frontend/unit/testingAngularUnitSpec.js [hunk 5]
```diff
@@ -1,14 +1,13 @@
...
       });
       expect(scope.menu.length).toBe(2);
-      expect(typeof scope.global).toBe("object");
+      expect(typeof scope.global).toBe('object');
```
Comment (emmyakin) [inline test/frontend/unit/testingAngularUnitSpec.js:11]: add _{ env: { "jasmine": true } }_ to .eslintrc

**Review by emmyakin — CHANGES_REQUESTED** [review]
make minor changes and update

File: test/frontend/unit/testingAngularUnitSpec.js (hunk 5 above)
Comment (andela-onnenanya) [reply test/frontend/unit/testingAngularUnitSpec.js:11]: Done

**Review by emmyakin — APPROVED** [review]

**Pull request merged**